doc/whats_new.rst

```diff
@@ -323,6 +323,14 @@ Enhancements
      (`#5540 <https://github.com/scikit-learn/scikit-learn/pull/5540>_`)
      By `Andreas Müller`_.
 
+   - When unpickling a scikit-learn estimator in a different version than the one
+     the estimator was trained with, a ``UserWarning`` is raised, see `the documentation
+     on model persistance
```

Comment:
Sorry for the pickiness but I believe it is persist**e**nce with an "e" and not an "a" ...